pkg/pod/pod.go

```diff
@@ -118,6 +118,8 @@ func (b *Builder) Build(ctx context.Context, taskRun *v1beta1.TaskRun, taskSpec
 	volumeMounts := []corev1.VolumeMount{binROMount}
 	implicitEnvVars := []corev1.EnvVar{}
 	featureFlags := config.FromContextOrDefaults(ctx).FeatureFlags
+	forbiddenEnvs := config.FromContextOrDefaults(ctx).Defaults.ForbiddenEnvs
+	fmt.Printf("\nForbidden envs: %#v\n", forbiddenEnvs)
```

Comment:
Is this for debugging or logging? If it's for logging, would it be better if we use logger here? similar to https://github.com/tektoncd/pipeline/blob/a01f1b13973b29746f933b670c0c370e54124bdc/pkg/reconciler/taskrun/taskrun.go#L99